generator/template/src/stories/index.stories.js

```diff
@@ -1,39 +1,12 @@
 <%_ if (hasEslintPluginImport) { _%>
 /* eslint-disable import/no-extraneous-dependencies */
 <%_ } _%>
-<%_ if (!csf) { _%>
-import { storiesOf } from '@storybook/vue'
-<%_ } _%>
 import { action } from '@storybook/addon-actions'
-<%_ if (hasBabel) { _%>
```

Comment:
Should not be removed.

Reply:
Why? does the `addon-links` plugin not function without babel?

Reply:
No, but we show jsx story if babel is enabled. And it uses addon links. So, this section is enabled only when babel is enabled

Reply:
Understood. My bad